bot/seasons/evergreen/minesweeper.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import typing
+from dataclasses import dataclass
+from random import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Client
+
+MESSAGE_MAPPING = {
+    0: ":stop_button:",
+    1: ":one:",
+    2: ":two:",
+    3: ":three:",
+    4: ":four:",
+    5: ":five:",
+    6: ":six:",
+    7: ":seven:",
+    8: ":eight:",
+    9: ":nine:",
+    10: ":keycap_ten:",
+    "bomb": ":bomb:",
+    "hidden": ":grey_question:",
+    "flag": ":triangular_flag_on_post:"
+}
+
+log = logging.getLogger(__name__)
+
+
+class CoordinateConverter(commands.Converter):
+    """Converter for Coordinates."""
+
+    async def convert(self, ctx, Coordinate: str) -> typing.Tuple[int, int]:
+        """Take in a Coordinate string and turn it into x, y"""
+        if not 2 <= len(Coordinate) <= 3:
+            raise commands.ArgumentParsingError()
+        value1 = Coordinate[0]
+        value2 = Coordinate[1:]
+        if not value2.isdigit():
+            raise commands.ArgumentParsingError()
+        x = ord(value1) - 97
+        y = int(value2) - 1
+        if (not 0 <= x <= 9) or (not 0 <= y <= 9):
+            raise commands.ArgumentParsingError()
+        return x, y
```

Comment:
This function is dense. Can you put spaces between chunks of code where it makes sense so this is easier to read?